client/src/pages/Dashboard/components/ChatList.js

```diff
@@ -0,0 +1,166 @@
+import ChatItem from './ChatItem';
+import EmojiIcon from '@material-ui/icons/SentimentSatisfiedOutlined';
+import PhotoLibraryIcon from '@material-ui/icons/PhotoLibraryOutlined';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+
+import ConversationService from '../../../services/conversationService';
+import { useAuth } from '../../../hooks/useAuth';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        height: '100%',
+        padding: theme.spacing(2),
+    },
+    header: {
+        background: 'white',
+        boxShadow: '0 2px 20px 0 rgba(88, 133, 196, .1)',
+        padding: "24px 30px"
+    },
+    onlineContainer: {
+        marginLeft: 20
+    },
+    onlineText: {
+        color: theme.palette.lightText
+    },
+    seeMoreButton: {
+        color: '#95A7C4',
+        '& svg': {
+            fontSize: 30
+        }
+    },
+    chatListContainer: {
+        height: "100%",
+        overflowY: 'auto',
+        overflowX: 'hidden',
+        padding: '20px 40px',
+        [theme.breakpoints.down('xs')]: {
+            padding: '20px 10px',
+        }
+    },
+    inputContainer: {
+        margin: "0 40px",
+        borderRadius: 5,
+        marginBottom: 20,
+        backgroundColor: '#F4F6FA',
+        [theme.breakpoints.down('xs')]: {
+            margin: "0 10px",
+        }
+    },
+    input: {
+        padding: '30px 25px',
+        fontWeight: 600,
+        fontSize: 14,
+        '&::placeholder': {
+            color: '#B1C3DF',
+            opacity: 1,
+        }
+    },
+    inputIcons: {
+        color: '#D1D9E6',
+    },
+}));
+
+export default function ChatList({ onlineStatus, backButtonCallback, selectedConversation }) {
+    const classes = useStyles();
+    const [messages, setMessages] = useState();
+    const auth = useAuth();
+
+    useEffect(() => {
+        async function getMessages() {
+            if (selectedConversation?.id) {
+                const messages = await ConversationService.getMessages(selectedConversation.id);
+                setMessages(messages);
+            }
+        };
+        getMessages();
+    }, [selectedConversation]);
+
+    const otherUser = useMemo(() => {
+        return selectedConversation?.users?.filter(user => user.id !== auth?.user?.id)[0];
+    }, [selectedConversation, auth]);
+
+    return (
+        <Grid className={classes.root} container direction="column" justify="space-between" wrap="nowrap">
+            <Grid item container className={classes.header} alignItems="center" justify="space-between" wrap="nowrap">
+                <Grid item>
+                    <Grid container spacing={0} alignItems="center">
+                        <Hidden mdUp>
+                            <Grid item >
+                                <IconButton onClick={backButtonCallback}>
+                                    <ArrowBackIosIcon />
+                                </IconButton>
+                            </Grid>
```

Comment:
This should be spaced a little from the username.

Reply:
fixed